Examples/apps/DestinationsExample/DestinationsExample/AppDelegate.swift

```diff
@@ -36,25 +37,55 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         // Add the Firebase destination plugin
         analytics?.add(plugin: FirebaseDestination(name: "Firebase"))
-
+        
+        //Add the AppsFlyer destination plugin
+        analytics?.add(plugin: AppsFlyerDestination(name: "AppsFlyer"))
+        
         return true
     }
-
+    
     // MARK: UISceneSession Lifecycle
-
+    
     func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
         // Called when a new scene session is being created.
         // Use this method to select a configuration to create the new scene with.
         return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
     }
-
+    
     func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
         // Called when the user discards a scene session.
         // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
         // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
     }
-
-
+    
+    //MARK: - Deep Link functionality
+    
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+        analytics?.continueUserActivity(userActivity)
+        return true
+    }
+    
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
+        analytics?.continueUserActivity(userActivity)
+        return true
+    }
+    
+    func application(_ application: UIApplication, open url: URL, sourceApplication: String?, annotation: Any) -> Bool {
+        analytics?.openURL(url)
+        return true
+    }
+    
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+        analytics?.openURL(url, options: options)
```

Comment:
Comment here to let the developer know this is needed for forwarding deep links for Appsflyer.